### ISSUE TITLE
[RE] How to train/val/test VESPCN

### ISSUE THREAD

**Issue by LoSealL** [LoSealL/VideoSuperResolution#10]
Questions:
a) When I trained the vespcn model on MCL-V, the psnr was around 23db but in the paper the theoretical value of psnr is 25 tested on vid4 dataset.I use command:
`python run.py --model=vespcn --dataset=mcl-v --threads=4 --memory_limit=1GB`

b) I felt confused about the training process of this module. 
> The code uses feature and label to calculate the loss, but I could not find where the feature and label are assigned. The feature is obtained by the forward process of vespcn network? And the loss is calculated by Huber loss according to the paper.

c) the verbosity and training_phase's roles are?

d) the input of vespcn should be several images for motion estimation, how can I know which frames are used as input to the network?

**Comment by LoSealL** [comment]
### About training process
`self.training_phase` is used for BN, True for training, False for testing and inferring.

`feature` and `label` is assigned in `Trainer`, the default trainer is `Trainer.VSR`. 

VESPCN is feed a clip of frames, which has shape [B, D, H, W, C] where B is batch and D is clip depth. We set depth in yaml config file. And the super resolved frame is the center of the clip, and that's why D must be an even number (3, 5, 7,...).

**Comment by LoSealL** [comment]
### About implementation difference to paper
The **motion estimation network** is exactly the same as paper's.
The **super-resolved network** is a little different while mine has more parameters.
The loss function may be a bit different but I thought that affects little to coverge.

**Comment by liweifdu** [comment]
Thank you, I understand!
